NuttX mainline is failing

It seems that some recent modification broke the building:
```
$ ./tools/configure.sh sim:nsh
$ make
/usr/bin/bash: line 1: /bin/llvm-ar: No such file or directory
make[3]: *** [/tmp/nuttxspace/apps/tools/Wasm.mk:112: /tmp/nuttxspace/apps/wasm/libwasm.a] Error 127
make[2]: *** [Makefile:53: /tmp/nuttxspace/apps/examples/hello_all] Error 2
make[1]: *** [Makefile:47: all] Error 2
make: *** [tools/LibTargets.mk:232: /tmp/nuttxspace/apps/libapps.a] Error 2
```

--- a/Application.mk
+++ b/Application.mk
@@ -268,4 +268,8 @@ distclean:: clean
 
 WASM_BUILD ?= n
 
-include $(APPDIR)/tools/Wasm.mk
+ifeq ($(WASM_BUILD),y)
+	ifneq ($(CONFIG_INTERPRETERS_WAMR)$(CONFIG_INTERPRETERS_WASM)$(CONFIG_INTERPRETERS_TOYWASM),)
+		include $(APPDIR)/tools/Wasm.mk
+	endif
+endif

